sbin/common/config_init.sh

```diff
@@ -366,6 +367,35 @@ function configDefaults() {
   # The OS kernel name, e.g. 'darwin' for Mac OS X
   BUILD_CONFIG[OS_KERNEL_NAME]=$(uname | awk '{print tolower($0)}')
 
+  # Determine OS full system version
+  local unameSys=$(uname -s)
+  local unameOSSysVer=$(uname -sr)
+  if [ "${unameSys}" == "Linux" ]; then
+    # Linux distribs add more useful distrib /etc/*-release file content
+    set +e
+    cat /etc/*-release > /dev/null
+    local cmdRc="$?"
+    if [ "${cmdRc}" == "0" ]; then
```

Comment:
How about:
```
if cat /etc/*-release` > /dev/null; then
```
instead of 
```
 cat /etc/*-release > /dev/null
    local cmdRc="$?"
    if [ "${cmdRc}" == "0" ]; then
```
Seems quite a bit simpler to read - at least to me.

Reply:
old habits! changed